pkg/cli/cli.go

```diff
@@ -389,55 +397,82 @@ func (c cli) buildRootCmd() *cobra.Command {
 
 // defaultCommand returns the root command without its subcommands.
 func (c cli) defaultCommand() *cobra.Command {
-	return &cobra.Command{
-		Use:   c.commandName,
-		Short: "Development kit for building Kubernetes extensions and tools.",
-		Long: fmt.Sprintf(`Development kit for building Kubernetes extensions and tools.
-
-Provides libraries and tools to create new projects, APIs and controllers.
-Includes tools for packaging artifacts into an installer container.
+	cmd := &cobra.Command{}
+	cmd.Use = c.cmdCfg.CommandName
+	if cmd.Use == "" {
+		cmd.Use = defaultCommandName
+	}
+	//Get default descriptions for root commands
+	sh, lo, ex := c.getRootCommandDefaults()
 
-Typical project lifecycle:
+	cmd.Short = c.cmdCfg.Short
+	if cmd.Short == "" {
+		cmd.Short = sh
+	}
 
-- initialize a project:
+	cmd.Long = c.cmdCfg.Long
+	if cmd.Long == "" {
+		cmd.Long = lo
+	}
 
-  %s init --domain example.com --license apache2 --owner "The Kubernetes authors"
+	cmd.Example = c.cmdCfg.Example
```

Comment:
```suggestion
	cmd.Example = fmt.Sprintf(c.cmdCfg.Example, cmd.Use)
```

Reply:
Users are setting this value, so `c.cmdCfg.Example` should be used without modification.